crates/bevy_render/src/lib.rs

```diff
@@ -366,7 +316,9 @@ fn extract(app_world: &mut World, render_app: &mut App) {
     // Note: We apply buffers (read, Commands) after the `MainWorld` has been removed from the render app's world
     // so that in future, pipelining will be able to do this too without any code relying on it.
     // see <https://github.com/bevyengine/bevy/issues/5082>
-    extract.apply_buffers(running_world);
+    extract.0.apply_buffers(running_world);
```

Comment:
Can we move this to the runner thread? It would avoid blocking the main world on commands. This is what #5082 is asking to do.

Reply:
we would need to send the extract stage through channels too, but I think it would be doable

Reply:
done.